Client/Connection.ts

```diff
@@ -23,6 +23,8 @@ export class Connection {
 		const headers: Record<string, string> = {
 			"Content-Type": "application/json; charset=utf-8",
 		}
+		const data = JSON.parse(window.sessionStorage.getItem("authData") ?? "{}")
+		this.#token = data.token
```

Comment:
i can see this failing if this client is used by the cde etc

can we have this in a try/catch?